README.md

```diff
@@ -0,0 +1,5 @@
+# Red Alert 2 mod for OpenRA
+
+[![Build Status](https://travis-ci.org/OpenRA/ra2.svg?branch=master)](https://travis-ci.org/OpenRA/ra2)
+
+See https://github.com/OpenRA/ra2/wiki for instructions.
```

Comment:
Sounds a bit harsh, IMHO, what about
`See [our wiki pages](https://github.com/OpenRA/ra2/wiki) for instructions on how to install and use this.`?